### DISCUSSION TITLE
Thanks for making my PC a living hell

### DISCUSSION THREAD

**Discussion by yotiscorp** [ionuttbara/windows-defender-remover#168] [General]
That script makes your PC the slowest thing on earth... I cant reset my computer since I ran that script Windows is telling me that it cant perform the reset....... every app takes 3h to load, games of now at 5FPS... My PC was in perfect conditions just wanted to remove Defender to get more specs. That thing is the complete opposite...  DO NOT USE THIS SCRIPT !

**Comment by BeegDeek** [comment]
It worked perfectly for me. You screwed up something

**Reply by axbender** [reply]
Same here. Works w/o hassle. 
On a Windows 10 installation (upgraded from Windows 7) with msmpeng.exe active when diffing two GNU installations turned the drives nearly inaccessible. I never hat this problem in Windows 7 with Defender running in the background. After using the tool, the problem's gone. Keep going!

**Comment by nook86** [comment]
is it fixes? for me in windows 11 startup menu dont work, i cant edit with text editor when i right click send to notepad
